helix-contract/contracts/mapping-token/v2/erc20-mapping-protocol/Erc20Sub2EthBacking.sol

```diff
@@ -30,11 +28,10 @@ contract Erc20Sub2EthBacking is Backing, DailyLimit, IBacking {
 
     event TokenLocked(uint256 transferId, bool isNative, address token, address sender, address recipient, uint256 amount, uint256 fee);
     event TokenUnlocked(uint256 transferId, bool isNative, address token, address recipient, uint256 amount);
-    event RemoteIssuingFailure(uint256 transferId, address mappingToken, address originalSender, uint256 amount, uint256 fee);
+    event RemoteIssuingFailure(uint256 refundId, uint256 transferId, address mappingToken, address originalSender, uint256 amount, uint256 fee);
     event TokenUnlockedForFailed(uint256 transferId, bool isNative, address token, address recipient, uint256 amount);
 
-    receive() external payable {
-    }
+    receive() external payable {}
```

Comment:
Why re-add `receive()`?

Reply:
this contract need to receive native token from weth by withdraw.

Reply:
Got it.